jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -689,7 +689,7 @@ public void addEventListener(EventListener listener)
     {
         _eventListeners.add(listener);
 
-        if (!(isStarted() || isStarting()))
+        if (!isRunning())
         {
             _durableListeners.add(listener);
```

Comment:
Is this `_durableListeners` dumped during server dump?
Either as the collection it is, or by decorating the normal listener collection with something indicating that it is a durable listener?
Seems like that would have been useful for debug of behaviors, especially considering that `!isRunning()` above is a subtle behavior that isn't documented in the apidoc.

Reply:
It would be good info to dump, but not easy to do without making things very verbose (ie repeating the listeners) as when dumping a listener it doesn't know if it is durable or not.